src/data/repositories/utils/AlertOutbreakMapper.ts

```diff
@@ -2,6 +2,13 @@ import { D2TrackerTrackedEntity } from "@eyeseetea/d2-api/api/trackerTrackedEnti
 import { alertOutbreakCodes } from "../consts/AlertConstants";
 import { getValueFromMap } from "./DiseaseOutbreakMapper";
 import { NotificationOptions } from "../../../domain/repositories/NotificationRepository";
+import { OutbreakDataType } from "../../../domain/entities/alert/OutbreakAlert";
+import { Id } from "../../../domain/entities/Ref";
+import {
+    RTSL_ZEBRA_ALERTS_DISEASE_TEA_ID,
+    RTSL_ZEBRA_ALERTS_EVENT_TYPE_TEA_ID,
+} from "../consts/DiseaseOutbreakConstants";
```

Comment:
No need to change, but to avoid long imports and dispersion of info, values can be grouped in the constants file by some criteria.